packages/algorand-js/src/lib/parsing.ts

```diff
@@ -110,7 +118,7 @@ function base64OrBase32 (args: string[], line: number): [string, EncodingType] {
 }
 
 /**
- * Description: returns encodingtype and string from words list
+ * returns encodingtype and string from words list
```

Comment:
Please update the description to make it more intuitive. what string are we returning and what is an encodingtype.

Reply:
yes, ok